run.sh

```diff
@@ -54,22 +54,40 @@ fi
 function doBackup {
 
     TIMESTAMP=`date +%Y-%m-%dT%H-%M-%S`
-    BACKUP_NAME="$TIMESTAMP.tar.gz"
+    BACKUP_NAME="$TIMESTAMP.tar"
     BUCKET_PATH="$BUCKET_PREFIX://$BUCKET/$BACKUP_NAME"
 
+    local algo=`echo $COMPRESSION | awk '{print tolower($0)}'`
+    case $algo in
+        gzip)
+            BACKUP_NAME="${BACKUP_NAME}.gz"
+            ;;
+        xz)
+            BACKUP_NAME="${BACKUP_NAME}.xz"
+            algo="${algo} -zT0"
+            ;;
+        zstd)
+            BACKUP_NAME="${BACKUP_NAME}.zst"
+            algo="zstd -zT0"
```

Comment:
Are these other apps installed on this image? zstd/xz?

Reply:
Very good point (and a bit of an oops from me); these aren't in the docker image, I've updated the Dockerfile now